frontend/src/metabase/entities/containers/EntityObjectLoader.jsx

```diff
@@ -106,7 +106,7 @@ export default class EntityObjectLoader extends React.Component {
   UNSAFE_componentWillReceiveProps(nextProps: Props) {
     if (
       nextProps.entityId !== this.props.entityId &&
-      this.props.entityId != null
+      nextProps.entityId != null
```

Comment:
Referring other questions using `{{#question_id}}` in the native query mode started failing because of this line which looks not correct. When we receive a new entityId which is not null we should fetch it. Before that, when users typed `{{#}}` and then for instance `{{#3}}` nextProps.entityId becomes `3` but `props.entityId` is null and this `if` state did not let fetch the new entity